src/server/routes/reviews.js

```diff
@@ -29,4 +30,44 @@ router.get('/:entityType/:bbid/reviews', async (req, res) => {
 	res.json(reviews);
 });
 
+router.post('/:entityType/:bbid/reviews', auth.isAuthenticated, async (req, res) => {
+	const editorId = req.user.id;
+	const {orm} = req.app.locals;
+
+	const {accessToken} = req.body;
+	const {review} = req.body;
+
+	let response = await cbHelper.submitReviewToCB(
+		accessToken,
+		review
+	);
+
+	// If the token has expired, we try to refresh it and then submit the review again.
+	if (response.message === 'The provided authorization token is invalid, expired, revoked, or was issued to another client.') {
```

Comment:
Is there not a specific error code we could use here instead? This seems a bit flimsy (would break if the error message ever changes)

Reply:
True. I'll replace it with a specific error code.